docs/docs/cmd/teams/funsettings/funsettings-list.md

```diff
@@ -51,3 +51,18 @@ m365 teams funsettings list --teamId 83cece1e-938d-44a1-8b86-918cf6151957
     allowGiphy,giphyContentRating,allowStickersAndMemes,allowCustomMemes
     1,moderate,1,1
     ```
+
+=== "Markdown"
+
+    ```md
+    # teams funsettings list --teamId "83cece1e-938d-44a1-8b86-918cf6151957"
+
+    Date: 5/7/2023
+
+    Property | Value
+    ---------|-------
+    allowGiphy | true
+    giphyContentRating | moderate
+    allowStickersAndMemes | true
+    allowCustomMemes | true
+     ```
```

Comment:
it seems this one is still wrongly formatted on the page 

![image](https://github.com/pnp/cli-microsoft365/assets/58668583/5b8308d1-43c7-4581-9da1-63ef79b7b87d)

It seems that the ```  is rendered in the table and we need an additional white line